prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml

```diff
@@ -20,6 +20,9 @@ presubmits:
       - name: cloudesf-testing-github-prow-service-account
         secret:
           secretName: cloudesf-testing-github-prow-service-account
+      - name: espv2-disk
+        persistentVolumeClaim:
```

Comment:
You'll need to want to mount this volume somewhere and make sure the job is using it (like you're doing with the `cloudesf-testing-github-prow-service-account` volume)